src/KKS_BrowserFolders_Hooks/Overlord.ImportHooks.cs

```diff
@@ -95,11 +95,14 @@ private static void AddFilesToConvList(string dir, ConvertChaFileScene instance,
             {
                 foreach (var file in Directory.GetFiles(dir))
                 {
-                    using (var reader = File.Open(file, FileMode.Open))
+                    if (file.EndsWith(".png"))
```

Comment:
```suggestion
                    if (file.EndsWith(".png", StringComparison.OrdinalIgnoreCase))
```
Case needs to be ignored.